src/command_modules/azure-cli-iot/HISTORY.rst

```diff
@@ -5,6 +5,7 @@ Release History
 0.3.10
 ++++++
 * Add support for manual-failover
+* Increment IotHub mgmt SDK requirement.
```

Comment:
This isn't customer facing and should be removed.

Reply:
removed